suites/pacific/upgrades/tier-2_scaleup_scaledown_after_upgrade_4x-to-5x-container.yaml

```diff
@@ -0,0 +1,598 @@
+# - Automation support for containerised upgrade from RHCS4 to RHCS5 cluster
+# - and test scale up and scale down of the cluster post upgrade
+# Cluster configuration: (conf/pacific/upgrades/tier-2_10node_cephadm_upgrade_scale.yaml)
+# --------------------------------------------------------------------------
+# Nodes:
+#    10-Node cluster(RHEL-8.3 and above)
+#    Initial cluster config - 3 MONS, 2 MDS, 1 MGR, 3 OSD and 2 RGW service daemon(s)
+#     Node1 - Mon, Mgr, Installer, OSD, alertmanager, grafana, prometheus, node-exporter
+#     Node2 - Mon, Mgr, OSD, MDS, RGW, alertmanager, node-exporter
+#     Node3 - Mon, OSD, MDS, RGW, node-exporter
+#     Node4 - Client
+#    Post upgrade
+#    Scale up cluster to 5 MONS, 5 MGRS, 4 RGWs, 4 MDSs, 2 NFSs, 2 ISCSIs and 10 OSD nodes
+#    Scale down cluster to initial configuration
+#
+# Test steps:
+# ---------------------------------------------------------------------
+# - Deploy containerised Nautilus Ceph cluster using CDN RPMS
+# - Run some I/O's
+# - Upgrade to Pacific using ceph-ansible and parallel run I/O's
+# - adopt to cephadm using ceph-adopt playbook
+# - Run some I/O's
+# - Scale up the cluster to 5 MONS, 5 MGRS, 4 RGWs, 4 MDSs, 2 NFSs, 2 ISCSIs and 10 OSD nodes
+# - Run some I/O's
+# - Scale down the cluster to its initial configuration
+# - Run some I/O's
+#
+#############################################################################
+tests:
+- test:
+    name: install ceph pre-requisites
+    module: install_prereq.py
+    config:
+      is_production: true
+    abort-on-fail: True
+
+- test:
+    name: ceph ansible install containerized rhcs 4.x from cdn
+    polarion-id: CEPH-83573588
+    module: test_ansible.py
+    config:
+      use_cdn: True
+      build: '4.x'
+      ansi_config:
+        ceph_origin: repository
+        ceph_repository: rhcs
+        ceph_repository_type: cdn
+        ceph_rhcs_version: 4
+        ceph_stable_release: nautilus
+        osd_scenario: lvm
+        osd_auto_discovery: False
+        ceph_stable_rh_storage: True
+        containerized_deployment: true
+        ceph_docker_image: "rhceph/rhceph-4-rhel8"
+        ceph_docker_image_tag: "latest"
+        ceph_docker_registry: "registry.redhat.io"
+        copy_admin_key: true
+        dashboard_enabled: True
+        dashboard_admin_user: admin
+        dashboard_admin_password: p@ssw0rd
+        grafana_admin_user: admin
+        grafana_admin_password: p@ssw0rd
+        node_exporter_container_image: registry.redhat.io/openshift4/ose-prometheus-node-exporter:v4.6
+        grafana_container_image: registry.redhat.io/rhceph/rhceph-4-dashboard-rhel8:4
+        prometheus_container_image: registry.redhat.io/openshift4/ose-prometheus:v4.6
+        alertmanager_container_image: registry.redhat.io/openshift4/ose-prometheus-alertmanager:v4.6
+        cephfs_pools:
+          - name: "cephfs_data"
+            pgs: "16"
+          - name: "cephfs_metadata"
+            pgs: "16"
```

Comment:
`pgs: 16` is this a requirement... if yes, could you please add a note in the top level document string.

Reply:
Not sure if this is a requirement. Removing it since it doesn't affect the scope of this test suite.